mlflow/utils/environment.py

```diff
@@ -5,48 +5,36 @@
 _conda_header = """\
 name: mlflow-env
 channels:
-  - defaults
   - conda-forge
 """
 
 
 def _mlflow_conda_env(
-    path=None,
-    additional_conda_deps=None,
-    additional_pip_deps=None,
-    additional_conda_channels=None,
-    install_mlflow=True,
+    path=None, additional_pip_deps=None, install_mlflow=True,
```

Comment:
Should we avoid removing `additional_conda_deps` and `additional_conda_deps` because it's a breaking change?

Reply:
We could depreate them in MLflow 1.1.8.0 and removed them in MLflow 1.19.0:
> `additional_conda_deps` and `additional_conda_deps` will be removed in the future.

Reply:
i think it's not an public API

Reply:
Offline discussion: we don't have to remove these arguments for now.